tests/contrib/django/test_django.py

```diff
@@ -1531,6 +1531,54 @@ def test_django_use_handler_resource_format_env(client, test_spans):
         assert out.startswith(b"Test success")
 
 
+@pytest.mark.parametrize(
+    "env_var,instrument_x",
+    [
+        ("DD_DJANGO_INSTRUMENT_DATABASES", "instrument_databases"),
+        ("DD_DJANGO_INSTRUMENT_CACHES", "instrument_caches"),
+        ("DD_DJANGO_INSTRUMENT_MIDDLEWARE", "instrument_middleware"),
+    ],
+)
+def test_enable_django_instrument_env(env_var, instrument_x):
+    """
+    Test that {env} enables instrumentation
+    """
+    with override_env({env_var: "true"}):
+        out = subprocess.check_output(
+            [
+                "ddtrace-run",
+                "python",
+                "-c",
+                "from ddtrace import config; assert config.django.%s" % instrument_x,
+            ],
+        )
+        assert out.startswith(b"")
+
+
+@pytest.mark.parametrize(
+    "env_var,instrument_x",
+    [
+        ("DD_DJANGO_INSTRUMENT_DATABASES", "instrument_databases"),
+        ("DD_DJANGO_INSTRUMENT_CACHES", "instrument_caches"),
+        ("DD_DJANGO_INSTRUMENT_MIDDLEWARE", "instrument_middleware"),
+    ],
+)
+def test_disable_django_instrument_env(env_var, instrument_x):
+    """
+    Test that {env} disables instrumentation
+    """
+    with override_env({env_var: "false"}):
+        out = subprocess.check_output(
```

Comment:
is there a reason `run_python_in_subprocess` doesn't work for these tests?

my only thinking is that it would make the test code look a little cleaner if you can do:

```python
<results> = run_python_code_in_subprocess("""
from ddtrace import config
assert not config.django.{}
""".format(instrument_x))

assert <results did not fail>
```

Reply:
Good point. I can use the `ddtrace_run_python_code_in_subprocess` fixture to clean up this code. The one drawback is that this fixture creates a new file, writes the test code to the file and then runs `subprocess.Popen` to execute the new test file. For each test this introduces an extra step of creating and writing to a file.

Reply:
yeah, that isn't a big deal. an extra file won't really impact the performance of the test.